2_observations.yml

```diff
@@ -134,3 +137,21 @@ targets:
       mgd_to_cms = 0.0438, cfs_to_cms = 0.0283,
       mg_to_cm = 4404.88, ft_to_m = 0.3048)
 
+ # reservoir surface area data
+ # You must download reservoir Monthly Shapefiles from here: 'http://umnlcc.cs.umn.edu/realsat/'.
+ # the corresponding shapefile folder is numbered according to the last digit of the ID.
+ # The ID can be found in the Base Shapefile.
+ # Unzip downloaded folders to 2_observations/in/realsat,
+ # such that the structure becomes:
+ # 2_observations/in/realsat/ID:
+      # in/realsat/573567
+      # in/realsat/009130
+  realsat_ids:
+    command: c(I(c(Pepacton = '009130', Cannonsville = '573567')))
```

Comment:
Very minor feedback here, but note that named vectors treat the names (here "Pepacton" and "Cannonsville") as argument names, and `c()` is a function in R, so this could be 
```yaml
  realsat_ids:
    command: c(Pepacton = I('009130'), Cannonsville = I('573567'))
```
No change necessary, as the result is the same, but wanted to point this out to make it clear what R is doing with named inputs to `c()` and how `I()` can be used with them. Same applies to lists, such as
```yaml
  realsat_ids:
    command: list(Pepacton = I('009130'), Cannonsville = I('573567'))
```